code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -73,6 +73,8 @@
 	if(pipe_vision_img)
 		qdel(pipe_vision_img)
 
+	SSair.pipenets_needing_rebuilt -= src
+

```

Comment:
Put this next to the other SSair accesses?

Reply:
fixed